<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UART_Plotter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(200,200,200);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>951</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="control_box">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Controls</string>
         </property>
         <widget class="QPushButton" name="start_btn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>61</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
         <widget class="QPushButton" name="stop_btn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>61</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
         <widget class="QDial" name="X_dial">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>170</y>
            <width>71</width>
            <height>91</height>
           </rect>
          </property>
          <property name="minimum">
           <number>-90</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
         <widget class="QDial" name="Y_dial">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>330</y>
            <width>71</width>
            <height>91</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>260</y>
            <width>41</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>X Axis</string>
          </property>
         </widget>
         <widget class="QLabel" name="X_label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>280</y>
            <width>61</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>410</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Y axis</string>
          </property>
         </widget>
         <widget class="QLabel" name="Y_label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>430</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
         <widget class="QPushButton" name="rX_btn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>310</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Reset X</string>
          </property>
         </widget>
         <widget class="QPushButton" name="rY_btn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>460</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Reset Y</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="PlotWidget" name="graphWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>1920</width>
           <height>1080</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
